utils/build.js

```diff
@@ -166,7 +166,7 @@ function generateBuildConfigs({formatToPath, baseDir, moduleName,
     buildConfigs.push({
       rollupConfig: {
         entry: entry || path.join(baseDir, 'src', 'index.js'),
-        plugins: [devReplacePlugin, ...basePlugins, babelPlugin],
```

Comment:
Personally, I'm concerned developers will be shipping dev mode because it's a pain to switch between the files and if so we should probably minify.